released/packages/coq-fiat-crypto/coq-fiat-crypto.0.0.15/opam

```diff
@@ -22,7 +22,7 @@ depends: [
   "coq-coqprime" {>= "1.2.0"}
   "coq-rewriter" {= "0.0.6"}
   "coq-rupicola" {= "0.0.5"}
-  "coq-bedrock2-compiler" {= "0.0.2"}
+  "coq-bedrock2-compiler" {= "0.0.3"}
```

Comment:
@JasonGross : please consider that such changes can break Coq Platform - as I said I took your changes as local patch to 0.0.2. This change to coq-fiat-crypto breaks this since I request an explicit version for coq-bedrock2 (0.0.2). Of course all in all it is agreeable since only 0.0.3 works without patches, but still this costs me 1/2 day (A Coq Platform CI run takes 6 hours).

Reply:
Ah, sorry about that.  What would you suggest in the future?  (I don't think we should keep version constraints that are known to be incompatible forever...)

Reply:
During the release phase of Coq Platform (like now) any changes in existing opam packages used in Coq Platform main or in the process of inclusion should be explicitly discussed upfront. I was aware of the PR but ddn't have time to review it. Just CCing me and noting that you change an existing package here would be sufficient. 

In cases like this were the existing opam package is simply wrong and can never work I would have taken measures upfront (like temporarily leaving the version of coq-bedrock2 and compiler open in the pick file).

Reply:
Or I could have done it in my local patch repo exactly as you did it here (create a new bedrock2 version).